download-robolectric-deps.sh

```diff
@@ -1,3 +1,5 @@
+set -e
```

Comment:
We were getting test failures in Robolectric because downloading these JARs failed but didn't stop the build at the `compile` stage.